### ISSUE TITLE
On Android, switching back to SN shows white window with red stripe.

### ISSUE THREAD

**Issue by ElCoyote27** [standardnotes/mobile#73]
On LineageOS 14.1and 15.1, when I switch back to SN, very often SN will show a white window with a red stripe at the top and no data.
When this happens, I have to close SN and re-open SN again (I have 2200 notes, this takes 30 seconds)
Is this behaviour expected?

**Comment by mobitar** [comment]
Not expected. Struggling to replicate this issue..

**Comment by ElCoyote27** [comment]
Hi Mobitar,
This is quite strange and I have also been struggling to reproduce the issue for the past few days. Would you rather close the issue and wait until I get a reliable reproducer to re-open?
Thanks,

**Comment by mobitar** [comment]
We can keep it open, so that it's easier to find for others who may be looking for something similar.

**Comment by ElCoyote27** [comment]
One thing that I just noticed:
I gave fingerprint lock a try and the issue wasn't happening anymore.
So I disabled fingerprint, re-enabled fingerprint and now the issue doesn't happen anymore.
This could be a red herrinf, though, as the 2.3.3 update with speedup might have solved the issue previously.

**Comment by mobitar** [comment]
Given the issue is seemingly random to begin with, you're likely to just suddenly encounter it at some point. But let me know if it continues going well for you.

**Comment by ElCoyote27** [comment]
I'm starting to wonder if this wouldn't be related to stuff that Android does under the hood.
A few days ago, I routinely updated LOS 15.1 (now running 20180731). This morning I noticed a variation:
Instead of a white window with a red stripe, I got a white window with a red strip with 'Notes' in the title (which I wasn't getting before). This happened when I brought back Notes from sleep the next morning.
During daytime, when I'm using my phone every few minutes, the issue doesn't occur anymore and Std Notes works fine.
